src/js/components/Header/Header.js

```diff
@@ -1,19 +1,86 @@
-import React from 'react';
-
+import React, { useEffect, useState } from 'react';
 import { Box } from '../Box';
+import { useForwardedRef } from '../../utils';
+
+const Header = React.forwardRef(({ sticky, ...rest }, ref) => {
+  const containerRef = useForwardedRef(ref);
+  const [stickyStyles, setStickyStyles] = useState();
```

Comment:
I'm wondering if we can simplify this further by removing `scrollUp` as state and just using `stickStyles`, setting it to `undefined` when we don't need it.

Reply:
good thinking I can try this